lib/index.js

```diff
@@ -39,44 +39,75 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   const accountPath = basePath + '/:account';
   const {baseUri} = bedrock.config.server;
 
+  /**
+   * @function createNewAccount
+   * @param {Request} req
+   * @param {Response} res
+   * @description
+   */
+  async function createNewAccount(req, res) {
+    const account = {
+      id: 'urn:uuid:' + uuid(),
+      email: req.body.email
+    };
+    if(!account.email) {
+      throw new BedrockError(
+        'No Email', 'InvalidEmailError', {
```

Comment:
We try to keep a very minimal set of "error types" (and they typically mirror those [found here](https://developer.mozilla.org/en-US/docs/Web/API/DOMException#Error_names)). That being said, we shouldn't have to check for `email` here -- as it should be done through the validator. It looks like we never got around to implementing the validator, though. So it looks like that's what needs to be done, see the TODO below:

```
// TODO: validate('services.account.postAccounts')
```

Reply:
I can switch to DataError instead

Reply:
I would normally say that's the right thing to do here -- but the validator will take care of it. So we should implement the validator instead and remove the special code.

Reply:
that is fairly easy to do where should I put the validator? in the file?

1. implement validator object as middle ware that calls next with a DataError
2. validate is a switch

Reply:
@aljones15 we have a module for that :)  `bedrock-validation` which among other things returns an express middleware that is used as shown in some of the commented code.  I'll share more with you about this in another context.

Reply:
ok cool I will look at bedrock-validation and add this to it. thanks man.